src/components/checkboxes/index.md.njk

```diff
@@ -0,0 +1,35 @@
+---
+title: Checkboxes
+description: Let users select one or more options by using the Checkboxes component.
+section: Components
```

Comment:
I think a lot of this metadata is redundant, at least for now. No harm in keeping aliases for now I guess, but `section`, `theme`, `status`, `department` and `copy` should all be removed.

Reply:
Fixed by 488acc1

Reply:
@36degrees Looks like we are using section.

```
<div class="app-c-content__header">
     <p class="app-c-content__section-heading">{{ section }}</p>
     <h1 class="app-c-content__heading heading-xlarge">{{ title }}</h1>
</div>
```
Should I put it back in or should this be coming from the file structure?

Reply:
@36degrees Just putting it back for now